server/emails/templates/MentionNotificationEmail.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import { Document } from "@server/models";
+import BaseEmail from "./BaseEmail";
+import Body from "./components/Body";
+import Button from "./components/Button";
+import EmailTemplate from "./components/EmailLayout";
+import Header from "./components/Header";
+import Heading from "./components/Heading";
+
+type InputProps = {
+  to: string;
+  documentId: string;
+  actorName: string;
+  eventName: string;
+  teamUrl: string;
+  mentionId: string;
+};
+
+type BeforeSend = {
+  document: Document;
+};
+
+type Props = InputProps & BeforeSend;
+
+/**
+ * Email sent to a user when someone mentions them in a doucment
+ */
+export default class MentionNotificationEmail extends BaseEmail<
+  InputProps,
+  BeforeSend
+> {
+  protected async beforeSend({ documentId }: InputProps) {
+    const document = await Document.unscoped().findByPk(documentId);
+    if (!document) {
+      return false;
+    }
+
+    return { document };
+  }
+
+  protected subject({ actorName, document }: Props) {
+    return `${actorName} mentioned you in "${document.title}"`;
+  }
+
+  protected preview({ actorName, eventName }: Props): string {
+    return `${actorName} ${eventName} you`;
```

Comment:
```suggestion
    return `${actorName} mentioned you`;
```

It's always the same 🤷